content/en/docs/references/_index.md

```diff
@@ -41,9 +41,8 @@ The CUE Go APIs in the main repo are organized as follows:
 - [**cue**](https://godoc.org/cuelang.org/go/cue):
```

Comment:
If you wanted to, these references to godoc.org could be changed to pkg.go.dev

Reply:
Good point. I changed the URL according to your advise.